e2e-tests/cypress/integration/appeal-submission-appeal-site-address/appeal-submission-appeal-site-address.js

```diff
@@ -120,16 +120,16 @@ Then(
 Then('the user is informed that {string}', (reason) => {
   switch (reason) {
     case 'Address Line 1 is required':
-      cy.confirmSiteAddressWasRejectedBecause('Enter a building and/or street');
+      cy.confirmSiteAddressWasRejectedBecause('Enter the first line of the address');
       break;
     case 'Address Line 1 has a limit of 60 characters':
       cy.confirmSiteAddressWasRejectedBecause(
-        'Building and/or street must be 60 characters or fewer',
+        'The first line of the address must be 60 characters or fewer',
       );
       break;
     case 'Address Line 2 has a limit of 60 characters':
       cy.confirmSiteAddressWasRejectedBecause(
-        'Building and/or street must be 60 characters or fewer',
+        'The first line of the address must be 60 characters or fewer',
```

Comment:
same as above ' should this be second line of address'

Reply:
'Line 141' still reads County is required, I think we can remove this case

Reply:
All of the requested changes are in the second commit to this PR:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1275/commits/1cf842f34300a8556ff775535bc224e6ea16b1f8

I've tagged a couple more of the scenarios with `@as-2483` in the FF, as these failed when I made the requested change, so do / should pass after the messaging was updated. Hopefully that helps narrow down which test cases to look at during FCQ.

I removed the two `case` statements around 'county is required' - thanks for flagging.